content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Mobile Session Replay Setup and Configuration
+kind: documentation
+description: Describes how to setup and configure Mobile Session Replay
+aliases:
+further_reading:
+    - link: '/real_user_monitoring/session_replay/mobile'
+      tag: Documentation
+      text: Mobile Session Replay
+    - link: '/real_user_monitoring/session_replay'
+      tag: Documentation
+      text: Session Replay
+---
+
+## Setup
+
+{{< tabs >}}
+{{% tab "Android" %}}
+
+All Session Replay SDK versions can be found in the [Maven Snapshots Repository][1].
+
+To set up Mobile Session Replay for Android:
+
+1. Make sure you've [setup and initialized the Datadog Android RUM SDK][2].
+2. In order to use the SNAPSHOTS in your Gradle dependencies, add this repository as a source in your repository configuration in your `build.gradle` file:
+
+   ```kotlin
+     repositories {
+          maven {
+           url ="https://oss.sonatype.org/content/repositories/snapshots"
+          }
+          }
+
+          dependencies {
+             implementation 'com.datadoghq:dd-sdk-android:1.19.0-sr-beta1-SNAPSHOT'
+             implementation 'com.datadoghq:dd-sdk-android-session-replay:1.19.0-sr-beta1-SNAPSHOT'
+             implementation 'com.datadoghq:dd-sdk-android-session-replay-material:1.19.0-sr-beta1-SNAPSHOT' // in case you need material support
+
+          }
+   ```
+
+3. Once the DD SDK and Session Replay SDK dependencies are imported, you can enable the feature when configuring the SDK:
+   ```kotlin
+      val config = Configuration.Builder(
+         logsEnabled = true,
+         tracesEnabled = true,
+         crashReportsEnabled = true,
+         rumEnabled = true
+      )
+      ...
+      .build()
+
+      ...
+
+      Datadog.initialize(context, credentials, config, trackingConsent) 
+      val sessionReplayConfig = SessionReplayConfiguration.Builder()
+         // in case you need material extension support
+         .addExtensionSupport(MaterialExtensionSupport()) 
+         .build()
+      val sessionReplayFeature = SessionReplayFeature(sessionReplayConfig)
+      Datadog.registerFeature(sessionReplayFeature)
+   ```
+
+[1]: https://oss.sonatype.org/content/repositories/snapshots/com/datadoghq/dd-sdk-android/
+[2]: /real_user_monitoring/android/?tab=kotlin
+
+{{% /tab %}}
+{{% tab "iOS" %}}
+
+
```

Comment:
placeholder: we'll need to add a link to the V2 docs as the intro for getting started, right @mariusc83 ?

Reply:
for both iOS and android setup

Reply:
@jamiemilstein exactly and this is the reason I mentioned above that we need to remove the part in the code snippets that explains how you first initialize the SDK

Reply:
these links were added by @rtrieu at line 33 above when she explains that you need to setup RUM first.